api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/PaymentCycleService.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.dhsc.htbhf.claimant.service;
+
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+import uk.gov.dhsc.htbhf.claimant.entity.Claim;
+import uk.gov.dhsc.htbhf.claimant.entity.PaymentCycle;
+import uk.gov.dhsc.htbhf.claimant.repository.PaymentCycleRepository;
+
+@Service
+@AllArgsConstructor
+public class PaymentCycleService {
+    private PaymentCycleRepository paymentCycleRepository;
+
+    public void createNewPaymentCycle(Claim claim) {
```

Comment:
This will need to accept a cycleStartDate and populate the cycleEndDate as well, but that can be in a subsequent commit.

Reply:
We're doing that work now for a future PR.